### PR TITLE
Pull cluster addon images from gcr.io

### PR DESCRIPTION
**Pull request by j3ffml**
Will do the same for e2e test images for #5007 after I finish uploading them.
cc @satnam6502, @zmerlynn

### CODE REVIEW

**Comment by satnam6502** [comment]
I can change the Makefiles for the Fluentd->ES, Fluentd->GCP, Elasticsearch and Kibana images in a different PR.

**Comment by satnam6502** [comment]
Oh, and I will also make gcr.io images for the node level images for Fluentd->ES and Fluentd->GCP

**Comment by mattmoor** [comment]
Awesome, thanks for the heads up.

File: cluster/addons/dns/skydns-rc.yaml.in [hunk 1]
```diff
@@ -21,7 +21,7 @@ desiredState:
         dnsPolicy: "Default"  # Don't use cluster DNS.
         containers:
           - name: etcd
-            image: quay.io/coreos/etcd:v2.0.3
+            image: gcr.io/google-containers/etcd:v2.0.3
```
Comment (zmerlynn) [inline cluster/addons/dns/skydns-rc.yaml.in]: We don't maintain this image, Core does. We should possibly keep it at quay unless Core wants to move it?

**Comment by zmerlynn** [comment]
LGTM except the quay/etcd question.

File: cluster/addons/dns/skydns-rc.yaml.in (hunk 1 above)
Comment (jlowdermilk) [inline cluster/addons/dns/skydns-rc.yaml.in]: Good point, reverted.

**Comment by zmerlynn** [comment]
LGTM

**Pull request merged**